packages/terra-tabs/src/TabPane.jsx

```diff
@@ -75,7 +78,7 @@ const TabPane = ({
     <div {...attributes} role="tab" className={paneClassNames}>
       {customDisplay}
       {customDisplay ? null : icon}
-      {customDisplay || isIconOnly ? null : <span className={cx('label')}>{label}</span>}
+      {customDisplay || isIconOnly ? null : <span className={classNames(cx('label'), theme.classNames)}>{label}</span>}
```

Comment:
I think you want the theme class name inside the cx function.
```suggestion
      {customDisplay || isIconOnly ? null : <span className={cx('label', theme.classNames)}>{label}</span>}
```

Reply:
Should I move theme.className inside cx or remove it as emily suggested above?

Reply:
Actually, you probably don't need to add the class here at all since it's added to the top of the component.